src/ring_theory/localization.lean

```diff
@@ -1016,6 +1016,48 @@ end
 
 end ideals
 
+section at_units
+variables (R) (S) (M)
+
+/-- The localization at a module of units is isomorphic to the ring -/
+noncomputable
+def at_units (H : ∀ x : M, is_unit (x : R)) : R ≃ₐ[R] S :=
+begin
+  apply alg_equiv.of_bijective (algebra.of_id R S),
+  split,
```

Comment:
This is not at all necessary, just pointing out some golfing opportunities in case you like them:
```suggestion
  refine alg_equiv.of_bijective (algebra.of_id R S) ⟨_, _⟩,
```